.moai/specs/SPEC-CURATION-001/spec.md

```
@@ -0,0 +1,43 @@
+### REQ-CUR-003 (Ubiquitous)
+manager-git의 tools SHALL NOT include `mcp__sequential-thinking__sequentialthinking` and `mcp__context7__*`.
+
+### REQ-CUR-004 (Ubiquitous)
+expert-refactoring의 tools SHALL NOT include `Agent` (leaf-node specialist).
+
+### REQ-CUR-005 (Ubiquitous)
+expert-debug의 tools SHALL NOT include `Agent` (leaf-node specialist).
+
+## Implementation Scope
+
+### Modified Files (Templates)
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)